scripts/rollup/rollup.config.js

```diff
@@ -14,7 +14,6 @@ const { rollup } = require('rollup');
 const replace = require('@rollup/plugin-replace');
 const typescript = require('@rollup/plugin-typescript');
 const { nodeResolve } = require('@rollup/plugin-node-resolve');
-const commonjs = require('@rollup/plugin-commonjs');
```

Comment:
`commonjs` plugin is no longer used here. Is it used elsewhere? If not, it should be removed from dependencies.